### DISCUSSION TITLE
Set npcstream

### DISCUSSION THREAD

**Discussion by NicolaSmaniotto** [Pryaxis/TShock#2978] [Q&A] [answered]
How can I set parameters such as `npcstream`? I'd like to lower its value because we often see players and mobs skipping around.

With the default server the [wiki](https://terraria.wiki.gg/wiki/Server#Server_config_file) says to write a file and pass `-config serverconfig.txt`, but TShock does not have that parameter in its [config.json fields](https://ikebukuro.tshock.co/#/config-file-descriptions).

**Comment by sgkoishi** [comment, marked as answer]
Use `-config` for vanilla serverconfig.txt, the TShock config.json is specified via `-configpath`.

**Reply by NicolaSmaniotto** [reply]
This does indeed work, but the documentation should be updated to reflect this.

[As it is currently worded](https://ikebukuro.tshock.co/#/command-line-parameters), it seems like `-config` works for TShock's own json configuration file, and `-configpath` for the whole folder of config files.
